saleor/graphql/shop/types.py

```diff
@@ -176,6 +181,10 @@ class Meta:
     def resolve_available_payment_gateways(_, _info, currency: Optional[str] = None):
         return get_plugins_manager().list_payment_gateways(currency=currency)
 
+    @staticmethod
+    def resolve_available_external_authentications(_, info):
```

Comment:
Should this field be public?

Reply:
Yeap, the same as we have public payment gateways :)